prisma/migrations/20240202103321_added_feedback_table/migration.sql

```diff
@@ -0,0 +1,12 @@
+-- CreateTable
+CREATE TABLE "feedback" (
+    "id" UUID NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "name" VARCHAR(128) NOT NULL,
+    "phone" VARCHAR(13) NOT NULL,
+    "email" VARCHAR(254),
+    "callAgain" BOOLEAN NOT NULL,
```

Comment:
🔴  what is 'callAgain' ? i think there should be something like 'callMe' or 'dontCallMe'

Reply:
I will rename it to "callMe"

Reply:
Email could be straight to 320 chars long. https://emaillistvalidation.com/blog/demystifying-email-validation-understanding-the-maximum-length-of-email-addresses/

Reply:
ok, thanks